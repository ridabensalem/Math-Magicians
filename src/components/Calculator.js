import '../calculator.css';

const Calculator = () => (
  <main className="calculator">
    <header className="calc-output">0</header>

    <div className="calc-grid">
      <section className="calc-numbers">
        <button id="btn-ac" className="btn" type="button">
          AC
        </button>
        <button id="btn-minus-plus" className="btn" type="button">
          +/-
        </button>
        <button id="btn-percentage" className="btn" type="button">
          %
        </button>
        <button id="btn-one" className="btn" type="button">
          1
        </button>
        <button id="btn-two" className="btn" type="button">
          2
        </button>
        <button id="btn-three" className="btn" type="button">
          3
        </button>
        <button id="btn-four" className="btn" type="button">
          4
        </button>
        <button id="btn-five" className="btn" type="button">
          5
        </button>
        <button id="btn-six" className="btn" type="button">
          6
        </button>
        <button id="btn-seven" className="btn" type="button">
          7
        </button>
        <button id="btn-eight" className="btn" type="button">
          8
        </button>
        <button id="btn-nine" className="btn" type="button">
          9
        </button>
        <button id="btn-zero" className="btn" type="button">
          0
        </button>
        <button id="btn-period" className="btn" type="button">
          .
        </button>
      </section>

      <section className="calc-operators">
        <button id="btn-four" className="btn" type="button">
          /
        </button>
        <button id="btn-four" className="btn" type="button">
          *
        </button>
        <button id="btn-four" className="btn" type="button">
          -
        </button>
        <button id="btn-four" className="btn" type="button">
          +
        </button>
        <button id="btn-four" className="btn" type="button">
          =
        </button>
      </section>
    </div>
  </main>
);

export default Calculator;
