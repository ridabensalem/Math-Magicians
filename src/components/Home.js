import React from 'react';

const Home = () => (
  <>
    <h1 className="header_magician">Welcome to Math Magician</h1>
    <p>
      The Math Magician Calculator is a remarkable tool that transcends the boundaries of t
      <br />
      raditional arithmetic computation. It possesses the magical ability to effortlessly perform
      complex mathematical operations wit
      h lightning speed and precision, leaving users awe-inspired and captivated.
    </p>
    <p>
      The Math Magician Calculator is not only highly efficient but also highly versatile,
      <br />
      capable of performing a vast range of mathematical operations with ease.
      Whether you need to perform basic arithmetic calculations,
      algebraic equations, trigonometric functions, or statistical analyses,
      this calculator has got you covered.
    </p>
  </>
);
export default Home;
