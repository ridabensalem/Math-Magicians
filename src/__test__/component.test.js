import { render } from '@testing-library/react';
import Home from '../components/Home';
import Quote from '../components/quote';
import Calculator from '../components/Calculator';

describe('Snapshot Test for Math Magicians project', () => {
  it('Snapshot Home rendering test', () => {
    const home = render(<Home />);
    expect(home).toMatchSnapshot();
  });
  it('Snapshot Quote rendering test', () => {
    const quote = render(<Quote />);
    expect(quote).toMatchSnapshot();
  });
  it('Snapshot Calculator rendering test', () => {
    const calc = render(<Calculator />);
    expect(calc).toMatchSnapshot();
  });
});
