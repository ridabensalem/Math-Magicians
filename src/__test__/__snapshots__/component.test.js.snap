// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test for Math Magicians project Snapshot Calculator rendering test 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <main
          class="calculator"
        >
          <header
            class="calc-output"
          >
             
          </header>
          <div
            class="calc-grid"
          >
            <section
              class="calc-numbers"
            >
              <button
                class="button_calculator"
                id="btn-ac"
                type="button"
              >
                AC
              </button>
              <button
                class="button_calculator"
                id="btn-minus-plus"
                type="button"
              >
                +/-
              </button>
              <button
                class="button_calculator"
                id="btn-percentage"
                type="button"
              >
                %
              </button>
              <button
                class="button_calculator"
                type="button"
              >
                1
              </button>
              <button
                class="button_calculator"
                id="btn-two"
                type="button"
              >
                2
              </button>
              <button
                class="button_calculator"
                id="btn-three"
                type="button"
              >
                3
              </button>
              <button
                class="button_calculator"
                id="btn-four"
                type="button"
              >
                4
              </button>
              <button
                class="button_calculator"
                id="btn-five"
                type="button"
              >
                5
              </button>
              <button
                class="button_calculator"
                id="btn-six"
                type="button"
              >
                6
              </button>
              <button
                class="button_calculator"
                id="btn-seven"
                type="button"
              >
                7
              </button>
              <button
                class="button_calculator"
                id="btn-eight"
                type="button"
              >
                8
              </button>
              <button
                class="button_calculator"
                id="btn-nine"
                type="button"
              >
                9
              </button>
              <button
                class="button_calculator"
                id="btn-zero"
                type="button"
              >
                0
              </button>
              <button
                class="button_calculator"
                id="btn-period"
                type="button"
              >
                .
              </button>
            </section>
            <section
              class="calc-operators"
            >
              <button
                class="button_calculator"
                id="btn-divide"
                type="button"
              >
                /
              </button>
              <button
                class="button_calculator"
                id="btn-multiply"
                type="button"
              >
                *
              </button>
              <button
                class="button_calculator"
                id="btn-subtract"
                type="button"
              >
                -
              </button>
              <button
                class="button_calculator"
                id="btn-add"
                type="button"
              >
                +
              </button>
              <button
                class="button_calculator"
                id="btn-equals"
                type="button"
              >
                =
              </button>
            </section>
          </div>
        </main>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <main
        class="calculator"
      >
        <header
          class="calc-output"
        >
           
        </header>
        <div
          class="calc-grid"
        >
          <section
            class="calc-numbers"
          >
            <button
              class="button_calculator"
              id="btn-ac"
              type="button"
            >
              AC
            </button>
            <button
              class="button_calculator"
              id="btn-minus-plus"
              type="button"
            >
              +/-
            </button>
            <button
              class="button_calculator"
              id="btn-percentage"
              type="button"
            >
              %
            </button>
            <button
              class="button_calculator"
              type="button"
            >
              1
            </button>
            <button
              class="button_calculator"
              id="btn-two"
              type="button"
            >
              2
            </button>
            <button
              class="button_calculator"
              id="btn-three"
              type="button"
            >
              3
            </button>
            <button
              class="button_calculator"
              id="btn-four"
              type="button"
            >
              4
            </button>
            <button
              class="button_calculator"
              id="btn-five"
              type="button"
            >
              5
            </button>
            <button
              class="button_calculator"
              id="btn-six"
              type="button"
            >
              6
            </button>
            <button
              class="button_calculator"
              id="btn-seven"
              type="button"
            >
              7
            </button>
            <button
              class="button_calculator"
              id="btn-eight"
              type="button"
            >
              8
            </button>
            <button
              class="button_calculator"
              id="btn-nine"
              type="button"
            >
              9
            </button>
            <button
              class="button_calculator"
              id="btn-zero"
              type="button"
            >
              0
            </button>
            <button
              class="button_calculator"
              id="btn-period"
              type="button"
            >
              .
            </button>
          </section>
          <section
            class="calc-operators"
          >
            <button
              class="button_calculator"
              id="btn-divide"
              type="button"
            >
              /
            </button>
            <button
              class="button_calculator"
              id="btn-multiply"
              type="button"
            >
              *
            </button>
            <button
              class="button_calculator"
              id="btn-subtract"
              type="button"
            >
              -
            </button>
            <button
              class="button_calculator"
              id="btn-add"
              type="button"
            >
              +
            </button>
            <button
              class="button_calculator"
              id="btn-equals"
              type="button"
            >
              =
            </button>
          </section>
        </div>
      </main>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Snapshot Test for Math Magicians project Snapshot Calculator should be equal when component is rendered 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <main
          class="calculator"
        >
          <header
            class="calc-output"
          >
             
          </header>
          <div
            class="calc-grid"
          >
            <section
              class="calc-numbers"
            >
              <button
                class="button_calculator"
                id="btn-ac"
                type="button"
              >
                AC
              </button>
              <button
                class="button_calculator"
                id="btn-minus-plus"
                type="button"
              >
                +/-
              </button>
              <button
                class="button_calculator"
                id="btn-percentage"
                type="button"
              >
                %
              </button>
              <button
                class="button_calculator"
                type="button"
              >
                1
              </button>
              <button
                class="button_calculator"
                id="btn-two"
                type="button"
              >
                2
              </button>
              <button
                class="button_calculator"
                id="btn-three"
                type="button"
              >
                3
              </button>
              <button
                class="button_calculator"
                id="btn-four"
                type="button"
              >
                4
              </button>
              <button
                class="button_calculator"
                id="btn-five"
                type="button"
              >
                5
              </button>
              <button
                class="button_calculator"
                id="btn-six"
                type="button"
              >
                6
              </button>
              <button
                class="button_calculator"
                id="btn-seven"
                type="button"
              >
                7
              </button>
              <button
                class="button_calculator"
                id="btn-eight"
                type="button"
              >
                8
              </button>
              <button
                class="button_calculator"
                id="btn-nine"
                type="button"
              >
                9
              </button>
              <button
                class="button_calculator"
                id="btn-zero"
                type="button"
              >
                0
              </button>
              <button
                class="button_calculator"
                id="btn-period"
                type="button"
              >
                .
              </button>
            </section>
            <section
              class="calc-operators"
            >
              <button
                class="button_calculator"
                id="btn-divide"
                type="button"
              >
                /
              </button>
              <button
                class="button_calculator"
                id="btn-multiply"
                type="button"
              >
                *
              </button>
              <button
                class="button_calculator"
                id="btn-subtract"
                type="button"
              >
                -
              </button>
              <button
                class="button_calculator"
                id="btn-add"
                type="button"
              >
                +
              </button>
              <button
                class="button_calculator"
                id="btn-equals"
                type="button"
              >
                =
              </button>
            </section>
          </div>
        </main>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <main
        class="calculator"
      >
        <header
          class="calc-output"
        >
           
        </header>
        <div
          class="calc-grid"
        >
          <section
            class="calc-numbers"
          >
            <button
              class="button_calculator"
              id="btn-ac"
              type="button"
            >
              AC
            </button>
            <button
              class="button_calculator"
              id="btn-minus-plus"
              type="button"
            >
              +/-
            </button>
            <button
              class="button_calculator"
              id="btn-percentage"
              type="button"
            >
              %
            </button>
            <button
              class="button_calculator"
              type="button"
            >
              1
            </button>
            <button
              class="button_calculator"
              id="btn-two"
              type="button"
            >
              2
            </button>
            <button
              class="button_calculator"
              id="btn-three"
              type="button"
            >
              3
            </button>
            <button
              class="button_calculator"
              id="btn-four"
              type="button"
            >
              4
            </button>
            <button
              class="button_calculator"
              id="btn-five"
              type="button"
            >
              5
            </button>
            <button
              class="button_calculator"
              id="btn-six"
              type="button"
            >
              6
            </button>
            <button
              class="button_calculator"
              id="btn-seven"
              type="button"
            >
              7
            </button>
            <button
              class="button_calculator"
              id="btn-eight"
              type="button"
            >
              8
            </button>
            <button
              class="button_calculator"
              id="btn-nine"
              type="button"
            >
              9
            </button>
            <button
              class="button_calculator"
              id="btn-zero"
              type="button"
            >
              0
            </button>
            <button
              class="button_calculator"
              id="btn-period"
              type="button"
            >
              .
            </button>
          </section>
          <section
            class="calc-operators"
          >
            <button
              class="button_calculator"
              id="btn-divide"
              type="button"
            >
              /
            </button>
            <button
              class="button_calculator"
              id="btn-multiply"
              type="button"
            >
              *
            </button>
            <button
              class="button_calculator"
              id="btn-subtract"
              type="button"
            >
              -
            </button>
            <button
              class="button_calculator"
              id="btn-add"
              type="button"
            >
              +
            </button>
            <button
              class="button_calculator"
              id="btn-equals"
              type="button"
            >
              =
            </button>
          </section>
        </div>
      </main>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Snapshot Test for Math Magicians project Snapshot Home rendering test 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1
        class="header_magician"
      >
        Welcome to Math Magician
      </h1>
      <p>
        The Math Magician Calculator is a remarkable tool that transcends the boundaries of t
        <br />
        raditional arithmetic computation. It possesses the magical ability to effortlessly perform complex mathematical operations wit h lightning speed and precision, leaving users awe-inspired and captivated.
      </p>
      <p>
        The Math Magician Calculator is not only highly efficient but also highly versatile,
        <br />
        capable of performing a vast range of mathematical operations with ease. Whether you need to perform basic arithmetic calculations, algebraic equations, trigonometric functions, or statistical analyses, this calculator has got you covered.
      </p>
    </div>
  </body>,
  "container": <div>
    <h1
      class="header_magician"
    >
      Welcome to Math Magician
    </h1>
    <p>
      The Math Magician Calculator is a remarkable tool that transcends the boundaries of t
      <br />
      raditional arithmetic computation. It possesses the magical ability to effortlessly perform complex mathematical operations wit h lightning speed and precision, leaving users awe-inspired and captivated.
    </p>
    <p>
      The Math Magician Calculator is not only highly efficient but also highly versatile,
      <br />
      capable of performing a vast range of mathematical operations with ease. Whether you need to perform basic arithmetic calculations, algebraic equations, trigonometric functions, or statistical analyses, this calculator has got you covered.
    </p>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Snapshot Test for Math Magicians project Snapshot Home should be equal when component is rendered 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1
        class="header_magician"
      >
        Welcome to Math Magician
      </h1>
      <p>
        The Math Magician Calculator is a remarkable tool that transcends the boundaries of t
        <br />
        raditional arithmetic computation. It possesses the magical ability to effortlessly perform complex mathematical operations wit h lightning speed and precision, leaving users awe-inspired and captivated.
      </p>
      <p>
        The Math Magician Calculator is not only highly efficient but also highly versatile,
        <br />
        capable of performing a vast range of mathematical operations with ease. Whether you need to perform basic arithmetic calculations, algebraic equations, trigonometric functions, or statistical analyses, this calculator has got you covered.
      </p>
    </div>
  </body>,
  "container": <div>
    <h1
      class="header_magician"
    >
      Welcome to Math Magician
    </h1>
    <p>
      The Math Magician Calculator is a remarkable tool that transcends the boundaries of t
      <br />
      raditional arithmetic computation. It possesses the magical ability to effortlessly perform complex mathematical operations wit h lightning speed and precision, leaving users awe-inspired and captivated.
    </p>
    <p>
      The Math Magician Calculator is not only highly efficient but also highly versatile,
      <br />
      capable of performing a vast range of mathematical operations with ease. Whether you need to perform basic arithmetic calculations, algebraic equations, trigonometric functions, or statistical analyses, this calculator has got you covered.
    </p>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Snapshot Test for Math Magicians project Snapshot Quote rendering test 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1>
        Math Quote 
      </h1>
      <q>
         Pure mathematics is, in its way, the poetry of logical ideas
      </q>
      <q>
         Without mathematics, there is nothing you can do
      </q>
      <q>
         Without mathematics, there is nothing you can calculate
      </q>
    </div>
  </body>,
  "container": <div>
    <h1>
      Math Quote 
    </h1>
    <q>
       Pure mathematics is, in its way, the poetry of logical ideas
    </q>
    <q>
       Without mathematics, there is nothing you can do
    </q>
    <q>
       Without mathematics, there is nothing you can calculate
    </q>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Snapshot Test for Math Magicians project Snapshot Quote should be equal when component is rendered 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1>
        Math Quote 
      </h1>
      <q>
         Pure mathematics is, in its way, the poetry of logical ideas
      </q>
      <q>
         Without mathematics, there is nothing you can do
      </q>
      <q>
         Without mathematics, there is nothing you can calculate
      </q>
    </div>
  </body>,
  "container": <div>
    <h1>
      Math Quote 
    </h1>
    <q>
       Pure mathematics is, in its way, the poetry of logical ideas
    </q>
    <q>
       Without mathematics, there is nothing you can do
    </q>
    <q>
       Without mathematics, there is nothing you can calculate
    </q>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
